test/integration/install_scripts_int_test.go

```diff
@@ -86,27 +86,25 @@ func (suite *InstallScriptsIntegrationTestSuite) TestInstall() {
 				argsWithActive = append(argsWithActive, "-c", cmd)
 			}
 
+			// Make the directory to install to.
```

Comment:
The uninstall integration test invokes an actual installer instead of mocking an install, and that invocation is based on this code with some improvements. I copied over those improvements here.